### PR TITLE
Renamed show methods to be more intuitive

### PR DESCRIPTION
**Pull request by rnystrom**
Minor change. Just a refactor, no logic change.

``` objc
// show the panels
- (void)showLeftPanelAnimated:(BOOL)animated;
- (void)showRightPanelAnimated:(BOOL)animated;
- (void)showCenterPanelAnimated:(BOOL)animated;
```

Reason being I nearly created more methods to choose whether or not to animate the changes. I assumed the <code>BOOL</code> was for choosing whether to show or hide the panels.

### CODE REVIEW

File: JASidePanels/Source/JASidePanelController.m [hunk 1]
```diff
@@ -422,7 +422,7 @@ - (void)_placeButtonForLeftPanel {
                 buttonController = [nav.viewControllers objectAtIndex:0];
             }
         }
-        if (!buttonController.navigationItem.leftBarButtonItem) {
+        if (!buttonController.navigationItem.leftBarButtonItem) {
```
Comment (rnystrom) [inline JASidePanels/Source/JASidePanelController.m:425]: Forgot to remove this whitespace change.

**Comment by l4u** [comment]
Could you flag the old methods as deprecated rather than removing them? Because this breaks the API and affects most of the users. Docs have to be updated as well.

**Comment by rnystrom** [comment]
Excellent points. Updating and committing.

**Comment by rnystrom** [comment]
Put old methods back in with deprecation flag with warning. 
Updated docs to use new method names.
Updated project settings for example to be compliant w/ Xcode 4.6.

**Pull request merged**